browser-extension/addTournesolRecommendations.js

```diff
@@ -78,6 +86,45 @@ const getTournesolComponent = (data) => {
     tournesol_link.append('learn more');
     inline_div.append(tournesol_link);
 
+    // Refresh button
+    refresh_button = document.createElement('button');
+    refresh_button.setAttribute('id', 'tournesol_refresh_button');
+    fetch(chrome.runtime.getURL('images/sync-alt.svg'))
+      .then(r => r.text())
+      .then(svg => refresh_button.innerHTML = svg);
+
+    refresh_button.className = 'tournesol_simple_button';
+    refresh_button.onclick = () => {
+      refresh_button.disabled = true;
+      loadRecommandations();
+    };
+    inline_div.append(refresh_button);
+    // Expand button
+    expand_button = document.createElement('button');
+    expand_button.setAttribute('id', 'tournesol_expand_button');
+    // A new button is created on each video loading, the image must be loaded accordingly
+    if(isExpanded){
+      fetch(chrome.runtime.getURL('images/chevron-up.svg'))
+        .then(r => r.text())
+        .then(svg => expand_button.innerHTML = svg);
+    }else{
+      fetch(chrome.runtime.getURL('images/chevron-down.svg'))
+        .then(r => r.text())
+        .then(svg => expand_button.innerHTML = svg);
+    }
```

Comment:
Remove some code duplication with a simple ternary operator

```suggestion
      fetch(chrome.runtime.getURL(isExpanded ? 'images/chevron-up.svg' : 'images/chevron-down.svg'))
        .then(r => r.text())
        .then(svg => expand_button.innerHTML = svg);
```